src/test/scala/uk/gov/hmcts/ccd/corecasedata/scenarios/CreateSSCSCaseData.scala

```diff
@@ -0,0 +1,132 @@
+package uk.gov.hmcts.ccd.corecasedata.scenarios
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import uk.gov.hmcts.ccd.util.{CcdTokenGenerator, PerformanceTestsConfig}
+
+import scala.concurrent.duration._
+import scala.util.Random
+
+
+object CreateSSCSCaseDataKJ extends PerformanceTestsConfig {
```

Comment:
KJ?

Reply:
This file is not needed under scenario folder so removed